src/hook/pauseResume.js

```diff
@@ -136,6 +146,41 @@ eg.module("pauseResume", ["jQuery"], function($) {
 	}
 
 	/**
+	 * Set a timer to delay execution of subsequent items in the queue.
+	 * And it internally manages "fx"queue to support pause/resume if "fx" type.
+	 *
+	 * @param {Number} An integer indicating the number of milliseconds to delay execution of the next item in the queue.
+	 * @param {String} A string containing the name of the queue. Defaults to fx, the standard effects queue.
+	 */
+	$.fn.delay = function(time, type) {
```

Comment:
@happyhj I agree it is more readable.My intension is to reduce confusion about third parameter because function interface is easily exposed. I'll consider more.. Is there any opinion @naver/egjs-dev ?